packages/core/documentation/src/components/DocumentationDisplay.vue

```diff
@@ -0,0 +1,270 @@
+<template>
+  <div class="documentation-display">
+    <div class="document-meta">
+      <div class="document-title meta-section">
+        <slot name="document-title">
+          <span>{{ fileName }}</span>.<span class="document-title-extension">md</span>
+        </slot>
+      </div>
+
+      <div
+        v-if="!hidePublishToggle"
+        class="document-status meta-section"
+      >
+        <span class="meta-label">{{ i18n.t('documentation.documentation_display.status_label') }}</span>
+        <KBadge
+          v-if="publishModel"
+          appearance="success"
+          is-bordered
+        >
+          {{ i18n.t('documentation.common.published') }}
+        </KBadge>
+        <KBadge
+          v-else
+          appearance="neutral"
+          is-bordered
+        >
+          {{ i18n.t('documentation.common.unpublished') }}
+        </KBadge>
+      </div>
+
+      <div
+        v-if="createdAt"
+        class="document-create meta-section"
+      >
+        <span class="meta-label">{{ i18n.t('documentation.documentation_display.added_label') }}</span>
+        <KBadge
+          :background-color="KUI_COLOR_BACKGROUND_NEUTRAL_WEAKER"
+          class="badge-modified"
+          :color="KUI_COLOR_TEXT_NEUTRAL"
+        >
+          {{ createdAt }}
+        </KBadge>
+      </div>
+
+      <div
+        class="document-display-actions"
+      >
+        <PermissionsWrapper
```

Comment:
question: where is this PermissionsWrapper component coming from? I don’t see an import

Reply:
fixed